Cargo.toml

```diff
@@ -44,7 +44,11 @@ regex-syntax = "0.8"
 ctrlc = "3.2"
 globset = "0.4"
 anyhow = "1.0"
-etcetera = "0.9"
+etcetera = "0.10"
+# Temporarily pin home to keep the MSRV of etcetera from increasing to 1.81.0.
+# This direct dependency can be removed after our MSRV is at least 1.81.0.
+# https://github.com/lunacookies/etcetera/releases/tag/v0.10.0
+home = "<0.5.11"
```

Comment:
This doesn't really work as desired.  If I run `cargo update` I get a duplicated `home` dependency, and the MSRV is still not respected:

```
├── etcetera v0.10.0
│   ├── cfg-if v1.0.0
│   └── home v0.5.11
├── home v0.4.2
```